### PR TITLE
Add OpenInWSL 1.5.0

### PR DESCRIPTION
**Pull request by Pololot64**
- [yes] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [yes] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [yes] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [yes] Have you tested your manifest locally with `winget install --manifest <path>`?
- [yes] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/24610)

### CODE REVIEW

File: manifests/o/opticos/openinwsl/1.5.0/opticos.openinwsl.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: opticos.openinwsl
+PackageVersion: 1.5.0
```
Comment (OfficialEsco) [inline manifests/o/opticos/openinwsl/1.5.0/opticos.openinwsl.installer.yaml:5]:
```suggestion
PackageVersion: 1.5
```

File: manifests/o/opticos/openinwsl/1.5.0/opticos.openinwsl.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: opticos.openinwsl
+PackageVersion: 1.5.0
```
Comment (OfficialEsco) [inline manifests/o/opticos/openinwsl/1.5.0/opticos.openinwsl.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.5
```

File: manifests/o/opticos/openinwsl/1.5.0/opticos.openinwsl.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: opticos.openinwsl
+PackageVersion: 1.5.0
```
Comment (OfficialEsco) [inline manifests/o/opticos/openinwsl/1.5.0/opticos.openinwsl.yaml:5]:
```suggestion
PackageVersion: 1.5
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
image
Remember to change the version folder name :)

**Comment by Pololot64** [comment]
@official esco. I am sorry for asking this but I am not a github genius :D. How to I change a folder name in a pull request? I approved all the changes but cannot figure it out.

**Comment by OfficialEsco** [comment]
How did you create the Manifests?
I would recommend using GitHub Desktop which i believe you are using?
* Open `GitHub Desktop`
* Go to the `OpenInWSL branch` (Probably your default by the looks of it)
* Click on `History`, right click on one of the files you created and select `Show in Explorer`
* Go back one folder
* Rename the the `1.5.0` to `1.5`
* Open each manifest and change the `PackageVersion` from `1.5.0` to `1.5`
* Open GitHub Desktop and click `Commit to Whatever branch`
* Click `Push`

**Comment by Pololot64** [comment]
@OfficialEsco I believed I have fixed the issues. The steps helped. Thanks

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**